File: SchoolManagementSystem/src/com/school/Result_Insert.java
package com.school;

import java.io.IOException;
import java.io.PrintWriter;

import javax.servlet.RequestDispatcher;
import javax.servlet.ServletException;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

public class Result_Insert extends HttpServlet {
	
	
	private static final long serialVersionUID = 1L;

	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		PrintWriter out = response.getWriter();
		response.setContentType("text/html");
		
		String studentName = request.getParameter("StudentName");
		String admissionNo = request.getParameter("AdmissionNo");
		String year = request.getParameter("Year");
		String classRoom = request.getParameter("classroom");
		String semester = request.getParameter("Semester");
		String sinhalaMarks = request.getParameter("SinhalaMarks");
		String buddhismMarks = request.getParameter("BuddhismMarks");
		String mathsMarks = request.getParameter("MathsMarks");
		String scinceMarks = request.getParameter("ScinceMarks");
		String englishMarks = request.getParameter("EnglishMarks");
		String historyMarks = request.getParameter("HistoryMarks");
		String citizenEducationMarks = request.getParameter("CitizenEducationMarks");
		String tamilMarks = request.getParameter("TamilMarks");
		String geographyMarks = request.getParameter("GeographyMarks");
		String healthandphysicaleducationMarks = request.getParameter("HealthandphysicaleducationMarks");
		String homeEconomicsMarks = request.getParameter("HomeEconomicsMarks");
		String aestheticsubjects = request.getParameter("Aestheticsubjects");
		String aestheticsubjectsMarks = request.getParameter("AestheticsubjectsMarks");
		String totalMark = request.getParameter("totalMark");
		String average = request.getParameter("average");
		
		
		boolean isTrue;
		
		isTrue = ResultDBUtil.studentresult(studentName,admissionNo,year,classRoom,semester,sinhalaMarks,buddhismMarks,mathsMarks,scinceMarks,
				englishMarks,historyMarks,citizenEducationMarks,tamilMarks,geographyMarks,healthandphysicaleducationMarks,homeEconomicsMarks,aestheticsubjects,aestheticsubjectsMarks,totalMark,average);
		
		if(isTrue == true) {
			RequestDispatcher dis = request.getRequestDispatcher("Result_Insert.jsp");//Connect home page
			dis.forward(request, response);
			out.println("<script type='text/javascript'>");
			out.println("alert('Student Result Insert Successfull');");
			out.println("</script>");
		}else {
			RequestDispatcher dis2 = request.getRequestDispatcher("unsuccess.jsp");//Connect home page
			dis2.forward(request, response);
		}
		
		
		
	}

}

File: SchoolManagementSystem/WebContent/JS/Result_Insert.js
/**

 * 
 */
function addNumbers()
                {
                        var SinhalaMarks = parseInt(document.getElementById("SinhalaMarks").value);                       
						var BuddhismMarks = parseInt(document.getElementById("BuddhismMarks").value);
						var MathsMarks = parseInt(document.getElementById("MathsMarks").value);
						var ScinceMarks = parseInt(document.getElementById("ScinceMarks").value);
						var EnglishMarks = parseInt(document.getElementById("EnglishMarks").value);
						var HistoryMarks = parseInt(document.getElementById("HistoryMarks").value);
						var CitizenEducationMarks = parseInt(document.getElementById("CitizenEducationMarks").value);
						var TamilMarks = parseInt(document.getElementById("TamilMarks").value);
						var GeographyMarks = parseInt(document.getElementById("GeographyMarks").value);
						var HealthandphysicaleducationMarks = parseInt(document.getElementById("HealthandphysicaleducationMarks").value);
						var HomeEconomicsMarks = parseInt(document.getElementById("HomeEconomicsMarks").value);
						var AestheticsubjectsMarks = parseInt(document.getElementById("AestheticsubjectsMarks").value);

                        var totalMark = document.getElementById("totalMark");
						var average=document.getElementById("average");
                        totalMark.value = SinhalaMarks + EnglishMarks + BuddhismMarks + MathsMarks + ScinceMarks + HistoryMarks + CitizenEducationMarks + TamilMarks + GeographyMarks + HealthandphysicaleducationMarks + HomeEconomicsMarks + AestheticsubjectsMarks;
						average.value = (SinhalaMarks + EnglishMarks + BuddhismMarks + MathsMarks + ScinceMarks + HistoryMarks + CitizenEducationMarks + TamilMarks + GeographyMarks + HealthandphysicaleducationMarks + HomeEconomicsMarks + AestheticsubjectsMarks) / 12;
                }